src/pypromice/postprocess/real_time_utilities.py

```diff
@@ -66,6 +66,10 @@ def get_latest_data(
         lin_reg_time_limit,
     )
 
+    if last_valid_index not in df_limited.index:
+        logger.info("No valid data limited period")
+        return None
+
     # Apply smoothing to z_boom_u
     # require at least 2 hourly obs? Sometimes seeing once/day data for z_boom_u
     df_limited = rolling_window(df_limited, "z_boom_u", "72H", 2, 1)
```

Comment:
Good point about the z_boom_u measurements. It would make sense to use more digits. I'll use 3 digits. 

It would definitely make sense to explore better methods for elevation measurements. 
**Accuracy:**
As I understand, our current accuracy is around ±10m, far from cm-level. As you suggested, we might be able to improve these measurements by incorporating fieldwork data from Jacob.
**Precision:**
Smoothing filters can help enhance the precision of the measurements. 
I have not investigated the data yet, but if you suggest <5cm *precision*, I can increase the number of digits to 2.

I am trying my best to steer between the terms accuracy vs precision 🙈😅.

PS: The BUFR schema uses 0.1m resolution for the elevation measurements. We are currently not using the z_boom_u values.

Reply:
Maybe it would be better to avoid hard assumptions about precision at this and the point and avoid hard rounding. The variables are anyway rounded in the BUFRVariables class and the eccodes export.
🤔